_layouts/blog.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en" class="h-100">
+  <head>
+    {% include header.html %}
+  </head>
+
+  <body class="d-flex flex-column h-100">
+    <header>
+      {% include navbar.html %}
+    </header>
+    <!-- Begin page content -->
+    <main role="main" class="flex-shrink-0">
+      <div class="container mt-4">
+	    <h2>{{ page.title }}</h2>
+	    <p><span class="font-weight-bold author">{{ page.author }}, </span><i> {{ page.date | date: "%b %d, %y" }}</i></p>
```

Comment:
change %y to %Y to render as 2018 instead of 18?

Reply:
:+1: